tests/pytorch/test_pytorch_model_export.py

```diff
@@ -817,35 +817,12 @@ def test_load_model_allows_user_to_override_pickle_module_via_keyword_argument(
         pickle_module=pickle,
     )
 
-    mlflow_torch_pickle_load = mlflow_pytorch_pickle_module.Unpickler
-
-    pickle_call_results = {"mlflow_torch_pickle_load_called": False}
-
-    def validate_mlflow_torch_pickle_load_called(*args, **kwargs):
-        pickle_call_results["mlflow_torch_pickle_load_called"] = True
-        return mlflow_torch_pickle_load(*args, **kwargs)
-
-    log_messages = []
-
-    def custom_warn(message_text, *args, **kwargs):
-        log_messages.append(message_text % args % kwargs)
-
-    with mock.patch(
-        "mlflow.pytorch.pickle_module.Unpickler"
-    ) as mlflow_torch_pickle_load_mock, mock.patch("mlflow.pytorch._logger.warning") as warn_mock:
```

Comment:
In this test, we check `Unpickler` is instantiated in `torch.load` but this approach no longer works in pytorch 1.9.0. See this PR for details.

https://github.com/pytorch/pytorch/pull/53139/

Reply:
As a workaround, we just check `torch.load` is called with the given `pickle_module`.